lib/trento/activity_logging/parser/metadata_enricher.ex

```diff
@@ -88,8 +88,9 @@ defmodule Trento.ActivityLog.Logger.Parser.MetadataEnricher do
        when activity in [:resource_tagging, :resource_untagging],
        do: {:ok, resource_id}
 
-  defp detect_enrichment(:host, {_, %{resource_id: id, operation: :saptune_solution_apply}}),
-    do: {:ok, id}
+  defp detect_enrichment(:host, {_, %{resource_id: id, operation: operation}})
+       when operation in [:saptune_solution_apply, :saptune_solution_change],
```

Comment:
Maybe we want to create an enum in operations, like `HostOperations` (as example, and `ClusterOperations` if needed in the future, etc) where we can list them all. So we don't need to be changing this.
What do you think?

Reply:
Glad you ask, @arbulu89 😄 take a look at this https://github.com/trento-project/web/pull/3499